clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -1276,10 +1276,21 @@ raw2trace_t::aggregate_and_write_schedule_files()
             vec.insert(vec.end(), keyval.second.begin(), keyval.second.end());
         }
     }
-    std::sort(serial.begin(), serial.end(),
-              [](const schedule_entry_t &l, const schedule_entry_t &r) {
-                  return l.timestamp < r.timestamp;
-              });
+    auto schedule_entry_comparator = [](const schedule_entry_t &l,
```

Comment:
Add comments both here and in invariant check's comparison that if one is changed the other must be changed, to avoid future inconsistency (or, share the code by exporting from raw2trace, but not sure how messy that gets).

Reply:
Done.